end-to-end-tests/cypress/integration/OSCAL/OSCAL_Load_Test.spec.js

```diff
@@ -26,7 +26,32 @@ describe('Test Loading System Security Plans', () => {
     cy.contains('This is the control implementation for the system.').should('be.visible')
   })
 })
-    
+
+describe('Test Editing System Security Plan Source', () => {
+  it('Successfully Edits SSP Source', () => {
+    cy.visit(Cypress.env('base_url')) 
+    cy.findByText('OSCAL Catalog Viewer').should('exist')
+    cy.get('button').first().click()
+    cy.contains('System Security Plan Viewer').click()
+    cy.findByText('OSCAL System Security Plan Viewer').should('be.visible')
+    cy.contains('Select OSCAL SSP').parent().click()
+    cy.contains('Enterprise Logging and Auditing System Security Plan').click()
+    cy.contains('Enterprise Logging and Auditing System Security Plan').should('be.visible')
+    const findKeystroke = Cypress.platform === 'darwin' ? '{meta}f' : '{ctrl}f'
+    cy.get('.monaco-editor textarea:first')
+      .type(findKeystroke).focused().type('Enterprise Logging and Auditing System Security Plan', {force: true})
+      .type('{esc}', {force: true}).focused().type('Even Better SSP', {force: true})
+    cy.contains('Save').click()
+    cy.contains('Even Better SSP').should('be.visible')
+    // Return to previous state
+    cy.get('.monaco-editor textarea:first')
+      .type(findKeystroke).focused().type('Even Better SSP', {force: true})
+      .type('{esc}', {force: true}).focused().type('Enterprise Logging and Auditing System Security Plan', {force: true})
+    cy.contains('Save').click()
+    cy.contains('Enterprise Logging and Auditing System Security Plan').should('be.visible')
```

Comment:
Should this also test that the page now displays the updated title?

Reply:
We [test that the first title change is displayed](https://github.com/EasyDynamics/oscal-editor-deployment/pull/54/files#diff-efb8b6b735e507f8c63bc03af8dbaab6bb14a692a0eb8ed5f380551d054213dbR45).  I guess you mean on the second save as well?